src/postgraphql/http/createPostGraphQLHttpRequestHandler.js

```diff
@@ -388,7 +388,7 @@ export default function createPostGraphQLHttpRequestHandler (options) {
       if (debugGraphql.enabled)
         debugGraphql(printGraphql(queryDocumentAst).replace(/\s+/g, ' ').trim())
 
-      const jwtToken = getJwtToken(req)
+      const jwtToken = getJwtToken(req, !!options.jwtSecret)
```

Comment:
Let’s use your old implementation from https://github.com/postgraphql/postgraphql/pull/433. I think that’s much cleaner. I don’t think it is too important to have the validation in https://github.com/postgraphql/postgraphql/blob/d4fd6a4009fea75dbcaa00d743c985148050475e/src/postgraphql/withPostGraphQLContext.ts#L121, but thanks so much for thinking about it!

We don’t have to delete the validation because it is still be useful for `withPostGraphQLContext` by itself. I don’t think missing the validation in this one case will be too big a deal. Do you agree?